debug-service-in-chrome.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+
+# Check if a port number is provided
+if [ -z "$1" ]; then
+  echo -e "\033[31mUsage: $0 <port>\033[0m"
+  exit 1
+fi
+
+PORT=$1
+
+# Get the PID of the process listening on the provided port
+PID=$(lsof -i:$PORT -t)
+
+# Check if the PID was found
+if [ -z "$PID" ]; then
+  echo -e "\033[31mNo process found listening on port $PORT.\033[0m"
+  exit 1
+else
+  # Send the SIGUSR1 signal to the process
+  kill -SIGUSR1 $PID
+  echo -e "\033[32mSent SIGUSR1 to process with PID $PID.\033[0m"
+fi
```

Comment:
We should handle multiple services being debugged by node. If one's is in use, would be better to show appropriate message accordingly
![image](https://github.com/opencrvs/opencrvs-core/assets/32668488/c1c9c5bf-b7cc-40b7-a466-a5cc9c4b9c9b)

Reply:
need a bit elaboration on this one.

Reply:
I think the current message seems to clear things up